README.md

```diff
@@ -25,16 +25,33 @@ bundle exec jekyll serve
 
 We are using the theme [Just the docs](https://github.com/pmarsceill/just-the-docs).
 
-Yo have to create a markdown file and follow the next structure:
+You have to create markdown files inside the folder struction, following the structure:
 
 ```
 ---
 layout: default
 title: A title
 parent: Name of the parent
 grand_parent: Name of the grand parent if needed
-has_child: true
+has_child: (bool)  
```

Comment:
leave it as true as the idea is to copy if you don't want child you directly delete the line